src/main/java/calculator/operation/ArithmeticOperation.java

```diff
@@ -0,0 +1,41 @@
+package calculator.operation;
+
+import calculator.exception.ErrorMessage;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public enum ArithmeticOperation {
+
+    PLUS("+", Math::addExact),
+    MINUS("-", Math::subtractExact),
+    TIMES("*", Math::multiplyExact),
+    DIVIDE("/", Math::floorDiv);
+
+    private final String expression;
+    private final BiFunction<Integer, Integer, Integer> function;
+
+    public static final Map<String, ArithmeticOperation> map = new HashMap<>();
+    static {
+        Arrays.stream(ArithmeticOperation.values())
+                .forEach(value -> map.put(value.expression, value));
+    }
+
+    ArithmeticOperation(final String expression, final BiFunction<Integer, Integer, Integer> function) {
+        this.expression = expression;
+        this.function = function;
+    }
+
+    public static ArithmeticOperation fromExpression(final String expression) {
+        if (map.containsKey(expression)) {
+            return map.get(expression);
+        }
+        throw new IllegalArgumentException(String.format(ErrorMessage.NOT_SUPPORTED_ARITHMETIC, expression));
```

Comment:
```suggestion
        if (!map.containsKey(expression)) {
            throw new IllegalArgumentException(String.format(ErrorMessage.NOT_SUPPORTED_ARITHMETIC, expression));
        }
        return map.get(expression);
```
이런 방식이 더 보편적이지 않을까요?
지금이야 return만 하기때문에 if문 내부가 길지 않아서 보지 흉하지 않지만, 로직이 조금만 추가되면, if문 내부에서 예외를 던져주는게 훨씬 깔끔할거 같아요.

코드에 통일성을 주기 위해서 위처럼 처리하는걸 권장해주고 싶네요 ㅎㅎ